openedx/frontend-app-course-authoring
# feat: add download template button to taxonomy list [FC-0036]

## Description
This PR adds a new button in the Taxonomy List to allow users to download a sample taxonomy template in the format used to import taxonomies.

![download-template](https://github.com/openedx/frontend-app-course-authoring/assets/849463/136bf531-3222-46fb-b1a0-64190d9d1c63)

## Supporting Information
* Part of: https://github.com/openedx/modular-learning/issues/114

## Testing instructions
- Go to the taxonomy list page (http://localhost:2001/taxonomy-list)
- Click on the `Download template` and then on the `CSV template` option
- Check if the `template.csv` is correctly saved.
- Click on the `Download template` and then on the `JSON template` option
- Check if the `template.json` is correctly saved.
____
Private-ref: [FAL-3532](https://tasks.opencraft.com/browse/FAL-3532)
